tests/PHPStan/Analyser/data/sscanf.php

```diff
@@ -1,6 +1,38 @@
 <?php
 
+namespace Sscanf;
+
 use function PHPStan\Testing\assertType;
 
-assertType('int|null', sscanf('20-20', '%d-%d', $first, $second));
-assertType('array|null', sscanf('20-20', '%d-%d'));
+function foo(string $s) {
+	assertType('int|null', sscanf($s, $s, $first, $second));
+	assertType('array|null', sscanf($s, $s));
```

Comment:
the 2 test-cases which existed before.. now with just a `string` format, because otherwise we would not get the same results

Reply:
done